<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" 	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">		<head>		<title>Jain PharmaBiotech - Lab-on-Chip and Microarrays</title>		<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />		<link rel="stylesheet" href="../base.css" type="text/css" />		<script src="http://www.google-analytics.com/urchin.js" type="text/javascript"> </script>
		<script type="text/javascript">
			_uacct = "UA-112726-1";
			urchinTracker();
		</script>
	</head>		<body>				<div id="navigation">			<a href="/">Home</a>			<a href="/reports/">Reports</a>			<a href="/publications/" class="current">Publications</a>			<a href="/lectures/">Lectures</a>			<a href="/contact.htm">Contact</a>		</div>				<div id="content">					<h1>Lab-on-Chip and Microarrays</h1>						<p><small><i>Pharmacogenomics 2001;2:73-77.</i><br />			<strong>CHI's 3rd Annual Conference: Lab-on-Chip and Microarrays, 22-24 January 2001, Zuerich, Switzerland.</strong><br />			Author &amp; Correspondance: <a href="mailto:jain@pharmabiotech.ch">Prof. K.K. Jain</a></small></p>						<h2>Abstract</h2>						<p>CHI's 3rd Annual conference on lab-on-chip and microarray technology  		covered the latest advances in this technology and applications in life  		sciences. Highlights of the meetings are reported briefly with emphasis  		on applications in genomics, drug discovery and molecular diagnostics.  		There was an emphasis on microfluidics because of the wide applications  		in laboratory and drug discovery. The lab-on-chip provides the  		facilities of a complete laboratory in a hand-held miniature device.  		Several microarray systems have been used for hybridisation and  		detection techniques. Oligonucleotide scanning arrays provide a  		versatile tool for the analysis of nucleic acid interactions and provide  		a platform for improving the array-based methods for investigation of  		antisense therapeutics. A method for analysing combinatorial DNA arrays  		using oligonucleotide-modified gold nanoparticle probes and a  		conventional scanner has considerable potential in molecular  		diagnostics. Various applications of microarray technology for  		high-throughput screening in drug discovery and single nucleotide  		polymorphisms (SNP) analysis were discussed. Protein chips have  		important applications in proteomics. With the considerable amount of  		data generated by the different technologies using microarrays, it is  		obvious that the reading of the information and its interpretation and  		management through the use of bioinformatics is essential. Various  		techniques for data analysis were presented. Biochip and microarray  		technology has an essential role to play in the evolving trends in  		healthcare, which integrate diagnosis with prevention/treatment and  		emphasise personalised medicines.</p>						<h2>Contents</h2>						<ul>				<li><a href="#1">Introduction</a></li>				<li><a href="#2">Microfluidics</a></li>				<li><a href="#3">Hybridisation and detection techniques</a></li>				<li><a href="#4">Genomic applications</a></li>				<li><a href="#5">Data analysis</a></li>				<li><a href="#6">Expert opinion and conclusions</a></li>				<li><a href="#7">Bibliography</a></li>			</ul>						<h2><a name="1">Introduction</a></h2>						<p>The use of miniaturised detection systems is now well established and 			recognised as the trend of the future for biomedical and biotechnical			applications. Biochip, defined as arrays of selected biomolecules			immobilised on a surface, is a broad term indicating the use of			microchip technology in molecular biology whereas microarray is a rapid			method of sequencing and analysing genes. Microfluidics is the analysis			of nanolitre amounts of fluids. At least 25 terms are used to describe			biochip and microarray technology (Jain 2000).</p>						<p>Innovations in technology, both in the industry and the academia,			were presented at Cambridge HealthTech Institute's 3rd Annual			Conference: "Lab-on-Chip and Microarrays", held on 22-24 January 2001,			at Swissotel Zuerich, Switzerland. This premier event in biochip			technology attracted over 400 participants. Highlights of the conference			that are relevant to application of microarray technology to genomics,			drug discovery and data analysis will be presented here. On the final			day, the lab-chip conference overlapped with the Cambridge Healthtech			conference on Integrated Bioinformatics, which followed the former			conference and is reported separately (Jain and Jain 2001).</p>						<h2><a name="2">Microfluidics</a></h2>						<p>Nick Hodge (Caliper Technologies, Mountain View, California, USA)			reviewed advances in microfluidic technology that have been developed			for a variety of genomic applications as well as continuous flow			biochemical and cell-based assays. Microfluidic devices have small			dimensions as grooves etched in glass, quartz or polymers and covered			with plates, which bond to form channels. The flow rate is usually slow,			commonly 1 nL moving through a distance of 1 mm/sec. Lab-on-a-chip			devices use the same manufacturing methods that are employed for			microchips in the computer industry called "microfabrication". Caliper			designs, manufactures, and commercialises LabChip® devices and systems			that enable experiments that ordinarily require laboratories full of			equipment and people to be conducted on a small chip. Caliper's LabChip			HP2100 Bioanalyzer is used for DNA sizing, DNA quantification, RNA			sizing and protein sizing in proteomics.</p>						<p>PAM microarray, a flow-through biochip technology, was presented by			Henk van Damme (PamGene, Boxtel, The Netherlands). It solves the			diffusion limitation problem of standard arrays by forcing the target			molecules through a porous array by force so that the hybridisation			process can be completed in a few minutes. The selectivity and			sensitivity of PAM microarray is such that it is feasible to			discriminate and analyse a point mutation after PCR. A 3-dimensional			Flow-Thru Chip was presented by Hongjun Yang (Gene Logic Inc			(Gaithersburg, Maryland, USA), is an assembly of thousands of			miniaturised microreactors that offer greater surface area and			manipulative microfluidics over which hybridisation can take place. It			is being developed for applications in expression profiling and related			genomic-based approaches to high-throughput screening. As a silicon			chip, it has less background noise than a glass chip, as there is no			cross talk between channels. Although there are advantages of 3-D			flow-through chips, the increase in capacity to hold held is countered by the			increased binding to the walls of the channels. Silicon chips have less			of a problem with fluid binding but more research needs to be done for			the flow-through chips.</p>						<p>Antonio Ricco (ACLARA Biosciences, Mountain View, CA, USA) described			disposable microfluidic arrays, often referred to as "lab-on-a-chip"			systems. This proprietary technology targets genomic applications such			as SNP typing, RNA/DNA analysis, sample preparation and high-throughput			pharmaceutical drug screening. The 96-plex microfluidic designs with			LabCard&trade; devices can be used for multiplexed gene expression analysis			using eTag&trade; technology as well as for multiplexed genotyping. A cost-			effective, single-use plastic lab card system is in development, which			uses electrokinetic effects to transport liquids through interconnected			microchannels. Cross-contamination of samples is avoided because the			chip is disposable. This approach reduces costs because of 20 to			100-fold reduction in the quantity of reagents required. Arteas&trade; technology			utilises proprietary microfluidics to automatically compensate for			evaporation from the assay well and thus reduce the consumption of a			target. It also could enable completing an initial screen of a promising			target sooner, and may reduce the cost of screenings.</p>						<p>A new approach to mass applications of microfluidics was described by			Stephen O'Connor (Nanostream Inc, Pasadena, California, USA). In this			modular approach, customised systems can be economically constructed in			a matter of hours rather than weeks, greatly reducing the product			development cycle. Various products include those for mixing two fluids,			for dividing a fluid into 64-different outputs, nanolitre sample			metering and nanolitre purification (Nanomodules&trade;).</p>						<h2><a name="3">Hybridisation and detection techniques</a></h2>						<p>Several hybridisation and detection techniques have been developed.			Mohammed Sohail (Oxford University, UK) described oligonucleotide-scanning			arrays. The basis of this is that antisense reagents inhibit			translation by binding with complimentary sites in mRNA and offer cheap			and simple alternative to gene knockout methods (Sohail and Southern			2000). They provide a versatile tool for the analysis of nucleic acid			interactions and provide a platform for improving the array-based			methods for investigation of antisense therapeutics.</p>						<p>Steve Hawkins (Genomic Solutions, Cambridgeshire, UK) compared the			variability and reproducibility of traditionally used techniques for			hybridisation and washing on standard glass slide microarrays with			automated techniques. Manual hybridisation under cover slip, currently			the most common method for processing microarrays, has its limitations.			Automated hybridisation involve uniform handling and offer many			advantages as they remove the problems of localised depletion and			washing conditions while reducing the hands-on time.</p>						<p>Sinchon Faigler (Compugen, Jamesburg, NJ, USA) described how the			company's LEADS transcriptosome prediction software platform has been			successfully applied in optimising the oligo-based chips. LEADS analyses			and clusters the sequences according to their gene association and			assembles the sequences in each cluster to produce a list of mRNA			transcripts. A system has been developed for designing the probes that			takes into account alternative splicing, which occurs in at least 35% of			human genes represented in the public EST databases and affect function			(Mironov et al 1999). It incorporates other probe selection criteria to			automatically choose optimal probe sequences.</p>						<p>Flavio Ortigao, (INTERACTIVA Biotechnoly, Ulm, Germany) described XNA			on Gold&trade;, which is INTERACTIVA's affinity array technology. The biochip			integrates stable thin and thick film technologies with robust			Streptavidin-based immobilisation, allowing the immobilization of			virtually any biomolecule. Target flexibility expands the horizons of			microarray technology beyond nucleic acid analysis, providing a			versatile platform for DNA, protein, and glycochips.</p>						<p>A method for analysing combinatorial DNA arrays using			oligonucleotide-modified gold nanoparticle probes and a conventional			flatbed scanner was presented by James Storhoff (Nanosphere Inc,			Evanston, Illinois, USA). Labelling oligonucleotide targets with			"high-density" nanoparticle rather than fluorophore probes significantly			alters the melting profiles of the targets from an array substrate. This			difference permits the discrimination of an oligonucleotide sequence			from targets with single nucleotide mismatches with a selectivity that			is over three times that observed for fluorophore-labelled targets. In			addition, when coupled with a signal amplification method based on			nanoparticle-promoted reduction of silver, the sensitivity of this			scanometric array detection system exceeds that of the analogous			fluorophore system by four orders of magnitude. This technology will			have a considerable impact on molecular diagnostics.</p>						<p>Planar Waveguide technology has been developed at Zeptosens			(Witterswil, Switzerland) and was presented by Gerhard Kresbach. It			overcomes current limitations sensitivity, signal-to-noise ratio and			quantitation obtained by microarray fluorescence signal readout. An up			to 100-fold increase in sensitivity compared to epifluorescence-based			readers enables high-sensitivity measurements of nucleic acids and			protein microarrays. Thus  limited amounts of biological material can be			investigated without target or signal amplification; thus avoiding bias			or non-linearity of the signal due to a biological or chemical			amplification process. An example of the performance of this approach is			demonstration of the expression of mRNA in 100 cell equivalents and			multianalyte analysis of interleukin 2, 4, and 6 in the pg/ml range. The			technology, Zepto&trade; MARK is expected to be available commercially during			the fourth quarter of 2001 and will be used for multiplexed immunoassays			and study of protein-protein interaction.</p>						<h2><a name="4">Genomic applications</a></h2>						<p>Tarif Awad of Affymetrix (Santa Clara, California, USA) described the			construction of the well-known GeneChip&trade; system combining integrated			instruments, arrays, and software for the massively parallel analysis of			thousands of transcripts per RNA sample. Developments in the technology			are continuing to increase the numbers of transcripts that can be			investigated per array, with the latest updates to human and murine			arrays containing probe sets to interrogate 12,500 sequences per array.			Current human coverage is 60,000 sequences over 5 arrays and murine			36,000 sequences over 3 arrays. GeneChip&trade; is a complete system with			software tools from other developers and the data can be analysed in			several ways. Absolute as well as multiple comparison analyses can be			done with good reproducibility. Several applications of this system were			discussed and it is considered to be ideal for target identification in			drug discovery. Affymetrix is conducting a study on myocardial			infarction in collaboration with Genentech (S. San Francisco,			California, USA). Over 40 genes and 11 metabolic pathways have been			identified. Identification of a subset of these have been correlated the			actions of the drug captopril. This system has also been used for cancer			classification based on microarray data. This has been found to be			particularly useful in distinguishing subtypes of acute lymphocytic			leukaemia (ALL) and acute myelogenous (AML) leukaemia, which is			difficult on the basis of morphology alone. Using clustering,			self-organising maps and algorithms, it is possible to distinguish			between B cell and T cell types of ALL.</p>						<p>New designs of microelectronic array-based systems for molecular			diagnostics, SNP analysis and pharmacogenomic applications were			presented by Michael Heller (Nanogen Inc, San Diego, California, USA).			An integrated system will include simple and inexpensive devices			designed to carry out sample preparation, cell lysis, DNA amplification			and multiplex hybridisation analysis. Additionally, Nanogen's			10'000-site chip is being developed for combinatorial ligand screening and drug			discovery applications.</p>						<p>Steven Lott (PerkinElmer NEN Life Sciences, Boston, Massachusetts,			USA) pointed out in his presentation that, contrary to the common			belief, "housekeeping genes" whose regulated and balanced expression			sustains metabolism, may not necessarily serve as reference standards			for data normalisation. Although, the Jurkat cell model did not reveal			any significant changes in housekeeping gene expression, variations in			the expression of certain housekeeping genes were observed by the use of			MICROMAX&trade; cDNA microarray technology on different biological models. For			example, differential expression of ribosomal genes was observed in			prostate and colorectal cancer models. Therefore, selection and use of			housekeeping genes for normalization of differential expression data			from various biological models should be approached with caution.</p>						<h2><a name="5">Data analysis</a></h2>						<p>The advent of high-throughput technologies has revolutionized gene			and protein expression analysis. With the considerable amount of data			generated by the different technologies using microarrays, it is obvious			that the reading of the information and its interpretation and			management through the use of bioinformatics is essential. The			prerequisite for the large-scale introduction of these technologies in			industrial research is to standardise procedures and to provide reliable			and reproducible results.</p>						<p>John Saldanha of the National Institute for Biological Standards and			Control (Potters Bar Herts, UK) gave a lecture on the standardisation of			biological assays. He pointed out those biological assays that rely on			the detection of differences in quantification of a biomolecule need to			be carefully validated and standardised. Standardisation of assays or			methods is essential before routine use of a method and can be achieved			by the use of reference reagents (run controls) in every assay run.			Validation of methods will depend on the parameters being analysed.			Qualitative assays may be considered as limit tests for the control of			impurities. In such cases, the main characteristics to be validated are			the specificity, detection limit, precision and robustness of the assay.			The International Conference on Harmonisation and the European			Pharmacopoeia has published guidelines for the validation of analytical			assays. The standardisation and validation of biological assays was			illustrated by reference to nucleic acid amplification technologies.			Validation and quality control of lab-chips and microarrays requires			quality control throughout the whole process. This includes preparation			of microarrays, extraction and amplification of nucleic acids, detection			of target sequence, and analysis of data. Standardisation is complex			because of the multiple elements using reference material, which is			difficult to establish.</p>						<p>Nikolai Kalnine (CLONTECH Inc, Palo Alto, California, USA) described			knowledge discovery tool for intrepretation of gene expression profiles.			"Semantic Link" is an Internet-based application that allows researchers			to rapidly navigate through, and derive useful information from, the 			growing volume of literature found in life science today. It contains a			map of biological semantic concepts (genes, diseases, etc.) that are			related to each other by the way that they are represented in the			prevailing scientific literature. "Semantic Link" is specifically			designed for helping scientists studying expression of genes with DNA			arrays on the stage of interpretation of the expression data in terms of			activation/suppression of biological processes.</p>						<p>Adesh Kaul (GeneData, Basel, Switzerland) spoke about the importance			of data quality assurance in large-scale gene expression studies.			GeneData has analysed the data output from several array technologies			and has designed technology-specific analysis tools that enable			scientists to monitor data quality and compare experiments on a company			wide scale. These tools have been implemented in the gene expression			data analysis software. GeneData Expressionist&trade;, a technology-independent			programme based on statistical analysis, can check the			reproducibility and sensitivity of data and detects problems in chip			production, probe preparation and image analysis. Protein expression			data analysis can be carried out with the software GeneData			Impressionist&trade;.</p>						<h2><a name="6">Expert opinion and conclusions</a></h2>						<p>A considerable amount of information was presented within three days.			The presentations were of high quality and all of these are not included			because of limitation of space. There was an open exchange of			information and discussion of controversial areas. The overall			impression gained from attendance at the conference is that the			microarray technologies have expanded considerably during the last few			years. The use of these technologies in the pharmaceutical industry has			not been as extensive as hoped for but it is expected to expand.			Biochip and microarray technology has an essential role to play in the			evolving trends in healthcare, which integrate diagnosis with			prevention/treatment and emphasise personalised medicines. The widest			clinical application is anticipated in molecular diagnostics. These			technologies have an important role to play in drug discovery, drug			safety and proteomics. Therefore, the pharmaceutical companies will be			making increasing use of these in the postgenomic era.</p>						<h2><a name="7">Bibliography</a></h2>						<ol>				<li>Jain KK. Biochips &amp; Microarrays: technology and commercial potential. Informa Pharmaceutical Publications, London, 2000.</li>				<li>Jain E, Jain KK. Integrated Bioinformatics: high-throughput interpretation of pathways and biology. Trends in Biotechnology 2001 May (in press).</li>				<li>Mironov AA, Fickett JW, Gelfand MS. Frequent alternative splicing of human genes. Genome Res 1999;9:1288-93.</li>				<li>Sohail M, Southern EM. Antisense arrays. Mol Cell Biol Res Commun 2000;3:67-72.</li>			</ol>							</div>				<div id="footer">			&copy; 2001 Jain PharmaBiotech. All Rights Reserved. 
		</div>			</body></html>