<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" 
	"http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">

<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">

	<head>
		<title>Proteomics: new technologies and their applications - Jain PharmaBiotech</title>
		<meta http-equiv="Content-Type" content="text/html; charset=iso-8859-1" />
		<link rel="stylesheet" href="../base.css" type="text/css" />
	</head>

	<body>
	
		<div id="navigation">
			<a href="/">Home</a>
			<a href="/reports/">Reports</a>
			<a href="/publications/" class="current">Publications</a>
			<a href="/lectures/">Lectures</a>
			<a href="/contact.htm">Contact</a>
		</div>
		
		<div id="content">
			
			<h1>Proteomics: new technologies and their applications</h1>
			
			<p><small><i>Drug Discovery Today 2001, 6:457-459.</i><br />
			Author &amp; Correspondance: <a href="mailto:jain@pharmabiotech.ch">Prof. K.K. Jain</a></small></p>
			
			
			<h2>Abstract</h2>
			
			<p>The increasing interest and activity in proteomics is evidenced by a 
			recent conference, which attracted a large attendance and was devoted 
			entirely to proteomic technologies. The 4th IBC conference on Proteomics 
			was held in Basel, Switzerland on 1-2 March 2001 and covered important 
			recent developments in this field.</p>
			
			<p>In the opening keynote address, Matthias Mann (MDS Proteomics, 
			Odense, Denmark) defined proteomics as the large-scale analysis of the 
			function of gene products not including genomics, bioinformatics and 
			mRNA studies, even though these are integrated as tools into proteomics. 
			A wider definition includes methods with genetic readout such as two-
			hybrid systems and structural genomics; a narrower definition includes 
			only large-scale approaches where actual proteins are studied 
			[<a href="#REF">1</a>].</p>
			
			<p>Mann described techniques for protein identification using MS in an 
			integrated system, including innovations of this technique such as MALDI 
			(matrix-assisted laser desorption/ionization) MS, nanoelectrospray 
			MS/MS, robotic spot excision, quadropole-TOF (time of flight), 
			microcapillary HPLC-MS/MS, and data management. Technology is now 
			maturing to identify all human proteins at high sensitivity and in a 
			reasonable throughput and more than 250 proteins have been identified so 
			far. Several proteomic technologies can now be used for investigating 
			cellular functions and to enhance classical biochemical approaches as a 
			part of functional proteomics.</p>
			
			<p>Twenty-five presentations were made at this conference. Only 
			highlights of the articles are presented here and other new developments 
			have been described in a special report on this topic 
			[<a href="#REF">2</a>].</p>
			
			
			<h2>Contents</h2>
			
			<ul>
				<li><a href="#00">Abstract</a></li>
				<li><a href="#01">Mass spectrometry</a></li>
				<li><a href="#02">Functional proteomics</a></li>
				<li><a href="#03">Protein arrays and high-throughput techniques</a></li>
				<li><a href="#04">Applications</a></li>
				<li><a href="#05">Conclusions</a></li>
				<li><a href="#REF">References</a></li>
			</ul>
			
			
			<h2><a name="01">Mass spectrometry</a></h2>
			
			<p>Mass spectrometry remains the workhorse for identification of 
			proteins but the standard techniques have limitations when it comes to 
			accurate quantification of proteins and peptides in complex mixtures. 
			Isotope-coded affinity tag (ICAT) peptide labelling is an approach that 
			combines accurate quantification and concurrent sequence identification 
			of individual proteins in complex mixtures [<a href="#REF">3</a>].</p>
			
			<p>Tim Nadler (Applied Biosystems, Foster City, CA, USA) presented the 
			optimized ICAT and validated the technique by demonstrating data using 
			real biological samples. The advantages of this technique are the 
			potential for automation and the relative quantification by MS. A 
			limitation of this approach is that it does not give information about 
			post-translational modification. Randall Nelson (Intrinsic Bioprobes, 
			Tempe, AZ, USA) described high-throughput protein characterization using 
			MS immunoassay (MSIA) combined with bioreactive probes and applied to 
			several body fluids. Examples presented included human cystatin C and 
			Beta-2 microglobulin screening of urine samples, which has clinical relevance 
			as biomarkers of inflammatory ailments. Throughput rates of ~100 samples 
			per hour were achieved. This technique is easy, fast and amenable to 
			robotics and protein profiling by MSAI can detect protein-protein and 
			protein-small-molecule interactions.</p>
			
			
			<h2><a name="02">Functional proteomics</a></h2>
			
			<p>Walter Blackstock (GlaxoSmithKline, Stevenage, UK) gave an excellent 
			overview of proteomic technologies within the broad field of genomics. 
			The focus was mainly on two areas:</p>
			
			<ul>
				<li>The use of proteomic technologies to characterize protein 
				complexes and pathways to create maps of spatial and temporal 
				expression of proteins in a 'virtual' cell; and</li>
				
				<li>The quantitative study of global changes in protein expression 
				in tissues to look for disease markers.</li>
			</ul>
			
			<p>These are usually protein-protein interactions and add value to the 
			target validation process.</p>
			
			<p>The MDS Proteomics (Odense, Denmark) approach to protein-protein 
			interactions was described by Ron Hendrickson and involves a fully 
			integrated programme. The three interacting components of this platform 
			are:</p>
			
			<ul>
				<li>PathMap, an automated pathway mapping system for identification 
				of crucial interactions and targets;</li>
				
				<li>BIND, a database for storage and interpretation of information 
				as well as for target selection; and</li>
				
				<li>LeadFinder for prioritization and quantification of hits whereby 
				information is generated for drug discovery.</li>
			</ul>
			
			<p>Other methods of examining protein-protein interactions were 
			discussed by Terry McCann (Perkin Elmer Life Sciences, Boston, MA, USA). 
			McCann and colleagues have designed a technology that uses green 
			fluorescent protein (GFP) and multibeam confocal imaging systems for 
			analysing the functions of proteins in living cells. The combination of 
			technologies enables 2-D PAGE (poly-acrylamide gel electrophoresis)/MS 
			approach in which the target protein is tagged with GFP enabling the 
			imaging of protein-protein interactions. This approach offers the 
			advantage that complex pathways of cell biology can be viewed without 
			destroying the cell.</p>
			
			<p>Donny Strosberg described the ongoing work at Hybrigenics (Paris, 
			France) where functional signalling and metabolic pathways are being 
			reconstructed on the basis of interacting proteins. Protein Interaction 
			Maps (PIMs) and corresponding annotated databases are then created using 
			HTS procedures and proprietary bioinformatic tools. In fact, the PIM of 
			<i>Helicobacter pylori</i> has already been published 
			[<a href="#REF">4</a>].</p>
			
			
			<h2><a name="03">Protein arrays and high-throughput techniques</a></h2>
			
			<p>Ian Humphrey-Smith (Universiteit Utrecht, Utrecht, The Netherlands) 
			described peptide, protein and antibody arrays as an integrated approach 
			to following human tissues in health and disease. The greater part of 
			the protein mass found within living cells is encoded by approximately 
			10% of genes. Most of the proteins, therefore, occur with a low 
			intracellular abundance and are beyond the reach of traditional 
			proteomic technologies such as 2-D gels and MS. Yet, knowledge of these 
			proteins is required for an understanding of the pathogenesis of 
			diseases. Antibody microarrays/biochips were proposed as a solution 
			because they enable coupling to sensitive detection systems, parallel 
			analysis and miniaturization. Differential display enables antibodies 
			with varying affinities to be employed on the same array. The following 
			presentation by Lucy Holt (MRC Laboratory of Molecular Biology, 
			Cambridge, UK) described the use of recombinant antibodies in proteomics 
			[<a href="#REF">5</a>]. Use of high-speed robotics is enabling millions 
			of antibody-antigen interactions to be screened simultaneously. This 
			approach obviates the need for phage selection and enables the 
			identification of differentially expressed proteins in natural extracts 
			as well as protein-protein interactions. The group is now exploring 
			miniaturization of these arrays.</p>
			
			<p>Since the post-genomic era, a need has been created for a large-scale 
			method to purify, identify and characterize novel and pharmacologically 
			relevant proteins. Stefan Schmidt (GPC Biotech, Martinsried, Germany) 
			presented technologies for high-throughput protein expression and 
			array-based characterization. Image analysis of arrays is done with its 
			Biochip Explorer technology. This method has been optimized to process 
			samples at a rate of 150 sec per sample with an average yield of at 
			least 1 Âµg and a purity of over 80%, which is adequate for many assays 
			in which thousands of proteins can be assessed for different 
			purposes.</p>
			
			<p>ProteinChip System (Ciphergen, Fremont, CA, USA), based on SELDI 
			(surface-enhanced laser desorption/ionization) is an alternative system 
			that can rapidly perform separation, detection and analysis of proteins 
			at femtomole levels directly from biological samples. William Rich of 
			Ciphergen presented both the ProteinChip 'benchtop' system and Tandem MS 
			system, which have several advantages over the 2-D gel method. These 
			include speed of detection (hours versus days), coverage of a broader 
			region of the proteome, small sample requirement (1 ml or 500 cells), 
			and combination of discovery of biomarkers and diagnostic assays in a 
			single system whereas MS is limited to the discovery of biomarkers only. 
			This has been exemplified by its use in the discovery of prostate cancer 
			biomarkers. With ProteinChip technology, it was possible to discriminate 
			between benign prostatic hypertrophy with bound PSA (prostate-specific 
			antigen) and cancer of the prostate with free PSA. Advantages of this 
			method over affinity LC-MS are similar to those over 2-D gels. 
			Additional features of the new system include the fact that it is a 
			non-expert, versatile benchtop system (in contrast to other expert, 
			dedicated affinity LC-MS systems) and can be automated for 
			high-throughput compatibility.</p>
			
			<p>Although the basic technologies of proteomics are well established, 
			key to success in their application to drug discovery is their ability 
			to handle a large number of protein targets in parallel. Malcolm Pluskal 
			(Proteome Systems, Woburn, MA, USA) discussed the Excise platform for 
			integrated spot cutting, digestion and spotting for MALDI-MS as well as 
			the applications of ProteoChip based on Chemical Printer, which employs 
			the piezo-electric microvolume liquid dispensing for chemical analysis 
			of the proteins resolved in a 2-D gel. After comparing with antigen and 
			antibody arrays, he pointed out these 2-D gels (GelChip) are authentic 
			protein chips although they are not predetermined arrays and the 
			location of the proteins in the array results from knowledge of the 
			charge and size of the proteins.</p>
			
			
			<h2><a name="04">Applications</a></h2>
			
			<p>The combination of DNA microarrays with proteomics has 
			revolutionalized molecular medicine, as these technologies reveal not 
			only gene regulation events involved in disease progression but also 
			targets for drug discovery and diagnostics 6. Julio Celis (Danish Centre 
			for Human Genome Research, Aarhus, Denmark) discussed the clinical 
			applications of proteomics with emphasis on premalignant lesions. 
			Proteomics combined with immunohistochemistry can be used for the study 
			of squamous cell carcinoma of the bladder. Protein markers specific for 
			epithelial cells of the tumour have been identified and antibodies 
			developed against these markers. Analysis by means of immunofluorescence 
			of cryostat sections of the tumour of patients with invasive disease is 
			based on proteomic data.</p>
	
			<p>Christian Rohlff (Oxford Glycosciences, Oxford, UK) described his 
			company's strategy for molecular dissection of disease with applications 
			to target discovery in human neurological disorders and cancer. A 
			region-specific CNS tissue expression database has been constructed as a 
			measure of disease specific to disease-relevant brain regions. This 
			links novel target mechanisms identified in CNS tissues to database 
			information relating to clinical, biological, image and annotation data. 
			An integrated analysis of human cerebrospinal fluid and CNS tissue, 
			which is being performed currently, could yield novel therapeutics or 
			support the rationale of existing drugs.</p>
	
			<p>Proteomics has also proved a valuable tool for the evaluation of 
			heart disease. John Weekes (Imperial College School of Medicine, Heart 
			Science Centre, Harefield, UK) described alterations in protein 
			expression in dilated cardiomyopathy. Specific proteins are 
			hyperubiquitinated in diseased hearts, and these proteins can be 
			purified by affinity chromatography and then identified using 2-D 
			PAGE/MALDI-TOF. The ubiquitin-proteosome pathway is therefore a possible 
			target for therapeutic intervention in heart disease.</p>
	
			<p>Michael Schrader (BioVision, Hannover, Germany) concluded the 
			programme by discussing peptidomics, i.e. the analysis of peptides from 
			biological sources and their application for the discovery of biomarkers 
			and drug candidates. Peptides are particularly suitable for diagnostics 
			because they result from regulatory/disease processes and soluble 
			molecules facilitate robust sample preparation, while peptide 
			fingerprints can be generated from a variety of different body fluids. 
			Therapeutic advantages of using peptides include convenience of 
			production and drug delivery.</p>
			
			
			<h2><a name="05">Conclusions</a></h2>
			
			<p>This conference reflected the considerable progress that has been 
			made in proteomics during the past few years. The technologies presented 
			are having a considerable impact on biomarker identification and drug 
			discovery. Although proteomics is being promoted as a separate industry, 
			it is in fact a set of technologies, which are being increasingly used 
			in combination with genomic technologies in the post-genomic era. The 
			conference was an excellent overview of the state-of-art proteomic 
			technologies and their applications, and provided a good forum for 
			discussion between the academia and the industry.</p>
			
			
			<h2><a name="REF">References</a></h2>
			
			<ol>
				<li>Pandey A. and Mann M. (2000) Proteomics to study genes and genomes.Nature, 405:837-846.</li>
				<li>Jain, K.K. (2001) <a href="/reports/proteomics/">Proteomics: Technologies and Commercial Opportunities</a> (4th edn), pp. 1-300, Jain PharmaBiotech.</li>
				<li>Gygi S.P. et al. (1999) Quantitative analysis of protein mixtures using isotope coded affinity tags.Nat. Biotechnol., 17:994-999.</li>
				<li>Rain J-C. et al. (2001) The protein-protein interaction map of Helicobacter pylori.Nature, 409:211-215.</li>
				<li>Holt L.J. et al. (2000) The use of recombinant antibodies in proteomics.Curr. Opin. Biotechnol., 11:445-449.</li>
				<li>Celis J.E. et al. (2000) Gene expression profiling: monitoring transcription and translation products using DNA microarrays and proteomics.FEBS Lett., 480:2-16.</li>
			</ol>
			
		</div>
		
		<div id="footer">
			&copy; 2001 Jain PharmaBiotech. All Rights Reserved. 
		</div>
	
	</body>
	
</html>
